avm/ptn/ai-platform/baseline/README.md

```diff
@@ -183,41 +283,105 @@ module baseline 'br/public:avm/ptn/ai-platform/baseline:<version>' = {
       "value": "aipbmax"
     },
     // Non-required parameters
-    "applicationInsightsSettings": {
+    "applicationInsightsConfiguration": {
       "value": {
         "name": "appi-aipbmax"
       }
     },
-    "containerRegistrySettings": {
+    "bastionConfiguration": {
+      "value": {
+        "disableCopyPaste": true,
+        "enabled": true,
+        "enableFileCopy": true,
+        "enableIpConnect": true,
+        "enableKerberos": true,
+        "enableShareableLink": true,
+        "name": "bas-aipbmax",
+        "networkSecurityGroupResourceId": "<networkSecurityGroupResourceId>",
+        "scaleUnits": 3,
+        "sku": "Standard",
+        "subnetAddressPrefix": "10.1.1.0/26"
+      }
+    },
+    "containerRegistryConfiguration": {
       "value": {
         "name": "craipbmax",
         "trustPolicyStatus": "disabled"
       }
     },
-    "keyVaultSettings": {
+    "keyVaultConfiguration": {
       "value": {
         "enablePurgeProtection": false,
         "name": "<name>"
       }
     },
-    "logAnalyticsSettings": {
+    "logAnalyticsConfiguration": {
       "value": {
         "name": "log-aipbmax"
       }
     },
-    "managedIdentitySettings": {
+    "managedIdentityConfiguration": {
       "value": {
         "name": "id-aipbmax"
       }
     },
-    "storageAccountSettings": {
+    "storageAccountConfiguration": {
       "value": {
         "allowSharedKeyAccess": true,
         "name": "staipbmax",
         "sku": "Standard_GRS"
       }
     },
-    "workspaceHubSettings": {
+    "virtualMachineConfiguration": {
+      "value": {
+        "adminPassword": "<adminPassword>",
+        "adminUsername": "<adminUsername>",
+        "enableAadLoginExtension": true,
+        "enableAzureMonitorAgent": true,
+        "enabled": true,
+        "encryptionAtHost": false,
+        "imageReference": {
+          "offer": "dsvm-win-2019",
```

Comment:
Is there a 2022 version?

Reply:
Yes, I wanted to use a different version than the default here to show that it's possible to supply a different image, however using the same version as the default is probably still good enough

Reply:
Updated